VCFv4.3.tex

```diff
@@ -162,7 +162,17 @@ \subsubsection{Information field format}
 \begin{itemize}
   \item A: The field has one value per alternate allele. The values must be in the same order as listed in the ALT column (described in section \ref{data-lines}).
   \item R: The field has one value for each possible allele, including the reference. The order of the values must be the reference allele first, then the alternate alleles as listed in the ALT column.
-  \item G: The field has one value for each possible genotype. The values must be in the same order as prescribed in section \ref{genotype-fields:genotype-ordering} (see \textsc{Genotype Ordering}).
+  \item G: The field has one value for each possible genotype.
+           Note that INFO tags declared as Number=G can have different number
```

Comment:
Should this block be reformatted to match the new 1 sentence per line formatting?